test/data/credentialsget.html

```diff
@@ -0,0 +1,10 @@
+<script>
+  var result;
+  navigator.credentials.get().then(function(data){
+    result = data;
+  });
+
+  function credentialsGetBlocked() {
```

Comment:
`result` will be `null` either way. You should initialize `result` to a non-null value. In this particular case the test should be fine because we're resolving immediately, but in other cases a test like this would have a race condition because `credentialsGetBlocked` could be called before the promise resolves